package.json

```diff
@@ -85,6 +85,7 @@
     "samples:build": "babel-node ./tooling/index.js build --only-samples",
     "samples:serve": "webpack-dev-server --env=samples",
     "samples:meetings": "npm run serve:package -- --env.package @webex/plugin-meetings --env.entry ./ciscospark.js",
+    "samples:devicemanager": "npm run serve:package -- --env.package @webex/plugin-devicemanager --env.entry ./ciscospark.js",
```

Comment:
I didn't catch this before, but we should rename this package `internal-plugin-device-manager` to be consistent with the other packages. Not in this change, but before we starting using it too much.